release_dashboard/analysis_options.yaml

```diff
@@ -1,6 +1,8 @@
+include: package:flutter_lints/flutter.yaml
+
 linter:
   rules:
     avoid_catches_without_on_clauses: true
```

Comment:
Are these rules in flutter.yaml? Can we remove them?

Reply:
I just saw another analysis_option file in the parent directory of `flutter/flutter` that the old release_dashboard was calling: https://github.com/flutter/flutter/blob/master/analysis_options.yaml

Should we add that file to the parent directory of Cocoon?

Reply:
No, flutter/flutter has a different style guide than what is recommended for flutter apps

Reply:
`avoid_catches_without_on_clauses: true` is used in the release_dashboard of 'flutter/flutter`, same for all the other ones, except `flutter_style_todos` which is not present in `flutter/flutter`.

Reply:
which ones you are suggesting to remove?

Reply:
This is what release dashboard in `flutter/flutter` uses:
```
    avoid_print: false
    only_throw_errors: false
    avoid_catching_errors: false
    avoid_catches_without_on_clauses: true
    prefer_relative_imports: true
    unawaited_futures: true
    ```

Reply:
Oh I see what you mean. I just verified, none of the rules here are present in the `recommended` options.

Reply:
I kept `flutter_style_todos` and `prefer_relative_imports: true`